styleguide/derek/global-theme/forms.ejs

```diff
@@ -77,6 +77,80 @@
   </form>
 </div>
 
+<div class="row half-padding">
+  <h5>Exposed Form</h5>
+  <div id="block-webform">
```

Comment:
Change this to `<div class="rs-webform-block rs-webform-embedded" id="block-webform-block">` to match what we have in production and then target off of `rs-webform-embedded`